clang/lib/CIR/CodeGen/CIRGenModule.cpp

```diff
@@ -3166,3 +3174,93 @@ LangAS CIRGenModule::getGlobalVarAddressSpace(const VarDecl *D) {
 
   return getTargetCIRGenInfo().getGlobalVarAddressSpace(*this, D);
 }
+
+mlir::cir::ConstArrayAttr CIRGenModule::EmitAnnotationString(StringRef Str) {
```

Comment:
All these methods should be camelBack and use "build" prefix -> "buildAnnotationString". https://llvm.github.io/clangir/GettingStarted/coding-guideline.html

Here and other methods added below.